source/server/config/http/fault.cc

```diff
@@ -5,8 +5,6 @@
 #include "common/config/filter_json.h"
 #include "common/http/filter/fault_filter.h"
 
-#include "api/filter/http/fault.pb.h"
```

Comment:
?

Reply:
This was an unnecessary include. The header was already including it.